modules/ROOT/pages/runtime-installation-task.adoc

```diff
@@ -114,7 +114,12 @@ Stopping Mule Enterprise Edition...
 Stopped Mule Enterprise Edition.
 ----
 
+== Enable Anypoint Monitoring 
+
+Optionally, you can install Anypoint Monitoring for cloud-managed supported versions of on-premises runtimes so you can monitor applications running on that server.
```

Comment:
Can we add a link in this paragraph to the other page?  I see it is in See Also but I think it would be intuitive for the user if there was also a link in this paragraph

Reply:
Sure.

Reply:
Thanks